static/scss/layout.scss

```diff
@@ -48,64 +50,56 @@ body {
   width: 100%;
   min-height: 100vh;
 
+  .persistent-drawer-open + .content {
+    padding-left: 240px;
+  }
+
   .content {
-    @include breakpoint(desktop) {
-      display: flex;
-      flex-direction: row;
-      margin: auto;
-      max-width: 1400px;
-    }
+    padding-top: 76px;
```

Comment:
Nit: This 76px value is also used in `drawer.scss`. Might be a good candidate for a constant. Your call